src/System.Collections/ref/System.Collections.cs

```diff
@@ -141,11 +142,13 @@ public sealed partial class KeyCollection : System.Collections.Generic.ICollecti
             System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
             public partial struct Enumerator : System.Collections.Generic.IEnumerator<TKey>, System.Collections.IEnumerator, System.IDisposable
             {
+                [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
+                [System.Diagnostics.CodeAnalysis.MaybeNullAttribute]
```

Comment:
I don't think we should preserve these attributes in the generic field, right?

I will fix `GenAPI` to not do so.

Reply:
Right; for exposed fields, yes, but not for private ones.

Reply:
Fixed.